test/puppetlabs/trapperkeeper/services/metrics/metrics_testutils.clj

```diff
@@ -0,0 +1,41 @@
+(ns puppetlabs.trapperkeeper.services.metrics.metrics-testutils
```

Comment:
I think pe-p-s-e used this namespace via a `test` package... so probably we're going to need a test package for tk-metrics too? We can maybe leave that for later though.